dependencycheck.groovy

```diff
@@ -4,17 +4,6 @@
 import hudson.util.VersionNumber
 import net.sf.json.*
 
-def listFromBintray() {
```

Comment:
This appears to be still up? https://bintray.com/api/v1/packages/jeremy-long/owasp/dependency-check

won't this stop old versions working?

It doesn't look like old versions were backfilled to github?

Reply:
I would just leave this for now, you could file an upstream issue asking them if they want to migrate their old versions off, but for now there's nothing for us to do here

Reply:
I’ve asked at https://github.com/jeremylong/DependencyCheck/issues/3367#issuecomment-836809728 although versions prior to the latest 6.1.6 are no longer supported so I’m not expecting any movement to happen.

https://github.com/jeremylong/DependencyCheck/blob/main/SECURITY.md

Reply:
Will anything here fail after bintray disappears if the code is left in?

Reply:
@timja Can we reopen this and merge?  Any references to bintray will be obsolete/dead as soon as the content on bintray.com is deleted, which is expected any time now.
Thanks

https://github.com/jeremylong/DependencyCheck/issues/3367#issuecomment-841868534

Reply:
Looks like the repo has disappeared now